src/gui/mainwindow.cpp

```diff
@@ -441,6 +441,7 @@ MainWindow::MainWindow(QWidget *parent)
 #endif
 
     m_propertiesWidget->readSettings();
+    m_ui->actionShowFiltersSidebar->setChecked(isFiltersSidebarVisible());
```

Comment:
From https://github.com/qbittorrent/qBittorrent/pull/10933#discussion_r311366621
> Chocobo1: Have you tried it in a clean installation? I mean when there are no config file and does it still work correctly?

Reply:
@AbeniMatteo 
Have you tested this instead? Piccirello never answered if he did it.

Reply:
Yes, it works fine.